provisionersdk/proto/provisioner.proto

```diff
@@ -272,6 +272,11 @@ message Role {
     string org_id = 2;
 }
 
+message RunningAgentAuthToken {
+    string agent_id = 1;
+    string token = 2;
+}
+
```

Comment:
When adding a new field, you need to increment `proto.CurrentVersion`. We tend to add a comment as well explaning the versioning history. See `provisionerd/proto/version.go`.